subcommand/control-plane/command.go

```diff
@@ -233,6 +216,14 @@ func (c *Command) realRun() error {
 		select {
 		case <-time.After(syncChecksInterval):
 			currentHealthStatuses = c.syncChecks(consulClient, currentHealthStatuses, serviceName, clusterARN, healthSyncContainers)
+		case watcherState := <-c.watcherCh:
+			c.log.Info("Consul server change detected. Reconfiguring consul client to connect to the newly discovered server(s)")
```

Comment:
We could include the server address here for clarity:

```suggestion
			c.log.Info("Switching to Consul server", "address", watcherState.Addr.String())
```